### PR TITLE
add plip

### PR DESCRIPTION
**Pull request by hadim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hadim** [comment]
ping @conda-forge/help-python

**Comment by hadim** [comment]
@marcelotrevisani could you review this please?

**Comment by fkaiserbio** [comment]
Please add the PharmAI organization (https://github.com/pharmai) to the maintainers.

File: recipes/plip/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - future
+    - numpy
+    - openbabel
+    - lxml
+
+test:
+  imports:
+    - plip
+  # commands:
+  #   - plip --help
```
Comment (hmaarrfk) [inline recipes/plip/meta.yaml:32]: did you want to test this?

Comment (hadim) [reply recipes/plip/meta.yaml:32]: yes but the command is not specified in the setup.py yet. I will be in the next version. I will enable it at that time.

**Comment by hadim** [comment]
ping @hmaarrfk

**Comment by hmaarrfk** [comment]
Unfortuantely, i cannot merge.

I ask that maybe you check that your library linked to the expected math libraries after you changed the falgs.

@conda-forge/staged-recipes this is likely ready to get merged. The remaining stuff seem like it can be fixed in the feedstock.

**Comment by hadim** [comment]
Sorry I am consufed about:

```
I ask that maybe you check that your library linked to the expected math libraries after you changed the flags.
```

Did I miss a comment?

**Comment by hmaarrfk** [comment]
no you didn't, but you removed some flags.

My experience is that those math libraries are sometimes optional and that they might get silently ignored.

For your own sake, check ;) but that can be fixed in the feedstock later so all good from the staged-recipes point of view.

**Comment by hadim** [comment]
I don't remember any math libraries. I just checked the original repo and all the deps are in the recipe.

**Comment by hadim** [comment]
Are you sure you're not talking about another recipe/repo? Anyway, if you can merge that would very nice :-)

And thank you for all by the way!

**Comment by hmaarrfk** [comment]
definietely an other recipe. thanks

**Comment by hadim** [comment]
LGTM @conda-forge/staged-recipes

**Comment by hadim** [comment]
@beckermr could you merge this please?

**Comment by hmaarrfk** [comment]
pro tip: seems like you care most about linux. 

I would say that for linux, patience can be gained by simply uploading to your own channels, and waiting for the standard conda-forge review process.

That way your own work isn't delayed by that of others ;)

**Pull request merged**